CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 IMPROVEMENTS:
  * client: Enable setting tags on Consul Connect sidecar service
 
+BUG FIXES:
+
+ * cli: Fixed an inconsistent order bug in `nomad alloc status` [[GH-6609](https://github.com/hashicorp/nomad/issues/6609)]
```

Comment:
How about "Make scoring column orders consistent in `nomad alloc status` CLI output" for a more direct statement about the change?

Reply:
That's much better thanks